src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift

```diff
@@ -65,7 +66,7 @@ struct FirstTicketValidationConsentViewModel {
 			// Bulletpoints in consent box
 			.section(
 				cells: [
-					.legalExtendedDataDonation(
+					.acknowledgement(
```

Comment:
did not know this cell type 😅

Reply:
I think it's the older one. It only supports one set of bullet points, which is enough in this case. With `legalExtendedDataDonation` there was some extra space at the bottom that we didn't need, that why I switched them.